sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -447,7 +460,7 @@ export class ServiceBusClient {
         maxAutoLockRenewalDurationInMs: options?.maxAutoLockRenewalDurationInMs,
         receiveMode,
         abortSignal: options?.abortSignal,
-        retryOptions: this._clientOptions.retryOptions,
+        retryOptions: { ...this._clientOptions.retryOptions, timeoutInMs: timeoutWithJitterInMs },
```

Comment:
How does this actually get into the link settings?

Reply:
If timeoutInMs is passed in when we create the receiver options, we set it in link properties in shared.ts above.

Reply:
Ah, I missed that. Still honing my TS skills.